metadata.yaml

```diff
@@ -29,11 +29,11 @@ peers:
 provides:
   kube-api-endpoint:
     interface: http
-  cluster-dns:
+  #cluster-dns:
     # kube-dns is deprecated. Its functionality has been rolled into the
     # kube-control interface. The cluster-dns relation will be removed in
     # a future release.
-    interface: kube-dns
+    #interface: kube-dns
```

Comment:
Can you remove the above 5 lines, rather than commenting the 2 lines out?

This comment saying "The cluster-dns relation will be removed in a future release." could be confusing for future readers of this file.